webservices/schemas.py

```diff
@@ -405,6 +435,14 @@ class CandidateHistoryTotalSchema(schemas['CandidateHistorySchema'], schemas['Ca
             'contributor_employer_text',
             'contributor_occupation_text',
         ),
+        'relationships': [
```

Comment:
And in that same vein of being duplicative, was it verified that the query supporting this relationship is only made once per request?  My concern is that if `relationship` is defined both here in the schema and on the model, that it may be doing the query twice?  Perhaps?  I don't know.  But with `schedule_a` and `schedule_b` for say the last cycle, that could be an extra join that is definitely not wanted.